demos/try17_AffineModel.py

```diff
@@ -0,0 +1,114 @@
+# %%
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+from cuqi.model import AffineModel, LinearModel
+from cuqi.experimental.mcmc import MH, LinearRTO
+from cuqi.distribution import Gaussian, LMRF, GMRF, JointDistribution
+from cuqi.geometry import Discrete, Continuous1D
+from cuqi.array import CUQIarray
+
+#%% Test problem 
+### y = Ax + b
+### Infer x, given the model A and b, and noisy observations of y
+
```

Comment:
It seems there is an empty figure when I run the script on my machine. Not sure why. For each case, linear and affine, there is the data/model outcome figure, the plot pair figure, and a third empty figure.

Reply:
Fixed.

Reply:
👍